test/issue.js

```diff
@@ -6,6 +6,8 @@ const zapier = require('zapier-platform-core');
 const App = require('../index');
 const appTester = zapier.createAppTester(App);
 
+const REPO = 'username/reponame' // CHANGE THIS
```

Comment:
Devs will need to change code to make the test pass. Wouldn't it be better if we keep it as an environment variable in `.env`?

Reply:
That was the value it was replacing, so I left it basically as is. 

I added copy in the `z/z` PR in the description to update this file, but I could also see setting the initial value to this repo so the tests run as written. 

Alternatively, we could make a test repo that we ignore, then we don't care if they open issues. 

What do you think?

Reply:
The main reason to move it was that it's not a secret value (which is typically what env is for). It's a per-test configuration, so it makes sense to set it that way.

Reply:
I personally prefer the original setup: having a `TEST_REPO` in the `.env` file, allowing the devs to set it to their personal repo without changing the code. @BrunoBernardino what do you think?

Reply:
I personally prefer the `.env` approach as well, but I guess @xavdid has a valid point with it not being secret and usually set once, so it's your call.